From 5b1834a41abd7ac2498bb9943ab973e07e2818d3 Mon Sep 17 00:00:00 2001
From: andryblack <blackicebox@gmail.com>
Date: Thu, 23 Apr 2020 21:57:23 +0300
Subject: [PATCH 2/3] fix include for cross-build

---
 Makefile         | 2 +-
 src/linux/main.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 2ca9a9f5..cb5e80e0 100644
--- a/Makefile
+++ b/Makefile
@@ -41,7 +41,7 @@ dirs-y = src
 cc-option=$(shell if test -z "`$(1) $(2) -S -o /dev/null -xc /dev/null 2>&1`" \
     ; then echo "$(2)"; else echo "$(3)"; fi ;)
 
-CFLAGS := -I$(OUT) -Isrc -I$(OUT)board-generic/ -std=gnu11 -O2 -MD -g \
+CFLAGS := -iquote $(OUT) -iquote src -iquote $(OUT)board-generic/ -std=gnu11 -O2 -MD -g \
     -Wall -Wold-style-definition $(call cc-option,$(CC),-Wtype-limits,) \
     -ffunction-sections -fdata-sections
 CFLAGS += -flto -fwhole-program -fno-use-linker-plugin
diff --git a/src/linux/main.c b/src/linux/main.c
index 4c5478eb..f6e96abd 100644
--- a/src/linux/main.c
+++ b/src/linux/main.c
@@ -4,7 +4,7 @@
 //
 // This file may be distributed under the terms of the GNU GPLv3 license.
 
-#include </usr/include/sched.h> // sched_setscheduler
+#include <sched.h> // sched_setscheduler
 #include <stdio.h> // fprintf
 #include <string.h> // memset
 #include <unistd.h> // getopt
-- 
2.24.2 (Apple Git-127)

